libbpf-tools/sigsnoop.c

```diff
@@ -63,6 +63,54 @@ static const char *sig_name[] = {
 	[29] = "SIGIO",
 	[30] = "SIGPWR",
 	[31] = "SIGSYS",
+	/**
+	 * manual signal(7)
+	 * The Linux kernel supports a range of 33 different real-time signals,
+	 * numbered 32 to 64. However, the glibc POSIX threads implementation
+	 * internally uses two or three real-time signals, and adjusts the
+	 * value of SIGRTMIN suitably (to 34 or 35).
+	 */
+#if CONST_SIGRTMIN == 35
+	[32] = "SIGRTMIN-3",
+	[33] = "SIGRTMIN-2",
+	[34] = "SIGRTMIN-1",
+else
```

Comment:
Let us `#elif CONST_SIGRTMIN == 34` here and later add `#else` branch like
```
error "Unsupported CONST_SIGRTMIN value"
```
to trigger a compilation error is CONST_SIGRTMIN is not 34/35.

Reply:
Modified in https://github.com/iovisor/bcc/compare/5d2579656c281d6718ae00abce1facca6749ce6b..7c66533ced0291d97a578889fbe6ed4126d554d5, thanks.